server/controllers/incidents.controller.js

```diff
@@ -45,45 +43,38 @@ export default {
     });
   },
 
-  postIncident: (req, res) => {
-    const typeOfIncident = util.incidentType(req.params.incidentType);
-    if (!typeOfIncident) {
-      return res.status(400).json({
-        status: 400,
-        message: 'Do you mean /red-flags or /interventions',
-      });
-    }
-    const report = util.newReport(req, typeOfIncident);
-
-    const result = validateIncident(report);
+  getInterventionById: (req, res) => {
+    const report = util.findById(incidents, req.params.id, 'intervention');
 
-    if (result.error) {
-      return res.status(400).json({
-        status: 400,
-        message: result.error.details[0].message,
+    if (!report) {
+      return res.status(404).json({
+        status: 404,
+        message: 'no report found, check the id or the incident type',
       });
     }
 
-    incidents.push(report);
+    return res.status(200).json({
+      status: 200,
+      data: [report],
+    });
+  },
+
+  postRecord: (req, res) => {
+    const newRecord = util.newReport(req);
+    incidents.push(newRecord);
 
     return res.status(201).json({
       status: 201,
       data: [{
-        id: report.id,
-        message: `created ${req.params.incidentType} record`,
+        id: newRecord.id,
+        value: newRecord,
```

Comment:
Hi, you can change the object key "value" here to "record"

Reply:
I have resolved this